src/components/SeqVizParamsModal.vue

```diff
@@ -179,9 +176,10 @@
                 if (is === was) {
```

Comment:
Not part of this PR, but I'm going to make a mental note of this being an interesting conditional.